invertedai/cosimulation.py

```diff
@@ -98,17 +113,30 @@ def ego_states(self):
         return [d for d, s in zip(self._agent_states, self._ego_agent_mask) if s]
 
     @property
-    def infractions(self) -> List[InfractionIndicators]:
+    def infractions(self) -> Optional[List[InfractionIndicators]]:
         """
         If `monitor_infractions` was set in the constructor,
         lists infractions currently committed by each agent, including ego agents.
         """
         return self._infractions
 
+    def get_infraction(self) -> Optional[Tuple[float, float, float]]:
```

Comment:
Can we just get rid of this function? I don't want to put anything here that's only used for our examples and we don't expect the customer to want to use. If the goal is to avoid duplication, we can have a shared module in the examples folder.